src/main/java/techreborn/blocks/BlockPlayerDetector.java

```diff
@@ -178,4 +178,11 @@ public IBlockState getStateFromMeta(int meta) {
 	public int getMetaFromState(IBlockState state) {
 		return typeNamesList.indexOf(state.getValue(TYPE));
 	}
+
+	@Override
+	public void addInformation(ItemStack stack, EntityPlayer playerIn, List<String> tooltip, boolean advanced) {
```

Comment:
The detect player (All) works but the other two modes do not they act the same as the (All) block